ata_db_models/models.py

```diff
@@ -46,14 +46,14 @@ class Event(SQLModel, table=True):
     # Data/attributes of HTML input and its form in JSON format. Only present if event_name == "change_form"
     # (read: https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0)
     # TODO probably optional and json
-    semistruct_form_change: Optional[str] = None
+    unstruct_event_com_snowplowanalytics_snowplow_change_form_1: Optional[str] = None
     # Data/attributes of HTML input and its form in JSON format. Only present if event_name == "focus_form"
     # (read: https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/focus_form/jsonschema/1-0-0)
     # TODO probably optional and json
-    semistruct_form_focus: Optional[str] = None
+    unstruct_event_com_snowplowanalytics_snowplow_focus_form_1: Optional[str] = None
```

Comment:
❓ asking because I don't know: do you want these to be string columns, or do you want them to be [postgres JSON columns](https://www.postgresql.org/docs/current/datatype-json.html)? I put it as string to start with, but we can change it to be more precise if desired.

Reply:
Yep!